### PR TITLE
Add Prometheus instrumenters and exporters

### PR DESCRIPTION
**Pull request by matt-mazzucato**
Add Prometheus instrumenters and exporters to pairing api

### CODE REVIEW

**Comment by matt-mazzucato** [comment]
Address #162

**Comment by rbino** [comment]
Add "Fix #162" in commit message

**Comment by rbino** [comment]
And add a line to CHANGELOG

File: apps/astarte_pairing_api/lib/astarte_pairing_api_web/metrics/prometheus_exporter.ex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2019 Ispirata Srl
```
Comment (bettio) [inline apps/astarte_pairing_api/lib/astarte_pairing_api_web/metrics/prometheus_exporter.ex]: 2020

**Pull request merged**